samples/detect.v2beta1.js

```diff
@@ -0,0 +1,854 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const projectId = process.env.GCLOUD_PROJECT;
+const sessionId = require('uuid/v1')();
+const common = require('@google-cloud/common');
+const structjson = require('./structjson.js');
+
+function createKnowledgeBase(projectId, displayName) {
+  // [START dialogflow_create_knowledge_base]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow client.
+  var client = new dialogflow.KnowledgeBasesClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
```

Comment:
❗️ **#BLOCKER**

The `// const projectId =` descriptions all still need to be updated

Original Comment: https://github.com/dialogflow/dialogflow-nodejs-client-v2/pull/144#discussion_r207396546

> "Let's use this:"
> "`// const projectId = 'ID of GCP project associated with your Dialogflow agent';`"